planet.rb

```diff
@@ -0,0 +1,17 @@
+class Planet
+
+  attr_reader :name, :color, :mass_kg, :distance_from_sun_km, :fun_fact
+
+  def initialize(name, color, mass_kg, distance_from_sun_km, fun_fact)
+    @name = name
+    @color = color
+    @mass_kg = mass_kg
+    @distance_from_sun_km = distance_from_sun_km
+    @fun_fact = fun_fact
+  end
+
+  def summary
+    return "#{@name}: #{@color}: #{@mass_kg}: #{@distance_from_sun_km}: #{@fun_fact}:"
```

Comment:
We didn't necessarily give a specific format that this summary should look, but the formatting of this is kind of weird? It ends up looking like this, with colons in a lot of weird places, which makes me suspect that you aren't sure how to format this text?

```
venus: sliver: 3030: 99922: its super far away:
```

Let me know if you have questions